.github/workflows/conformance.yaml

```diff
@@ -110,94 +109,6 @@ jobs:
           - webhooks
     needs: prepare-images
     name: ${{ matrix.k8s-version.name }} - ${{ matrix.config.name }} - ${{ matrix.tests }}
-    steps:
-      - name: Checkout
-        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
-      - name: Setup caches
-        uses: ./.github/actions/setup-caches
-        timeout-minutes: 5
-        continue-on-error: true
-        with:
-          build-cache-key: run-conformance
-      - name: Setup build env
-        uses: ./.github/actions/setup-build-env
-        timeout-minutes: 10
-      - name: Create kind cluster
-        shell: bash
-        run: |
-          set -e
-          export KIND_IMAGE=kindest/node:${{ matrix.k8s-version.version }}
-          make kind-create-cluster
-      - name: Download kyverno images archive
-        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # v3.0.2
-        with:
-          name: kyverno.tar
-      - name: Load kyverno images archive in kind cluster
-        shell: bash
-        run: |
-          set -e
-          make kind-load-image-archive
-      - name: Install kyverno
-        shell: bash
-        run: |
-          set -e
-          export USE_CONFIG=${{ join(matrix.config.values, ',') }}
-          make kind-install-kyverno
-      - name: Wait for kyverno ready
-        uses: ./.github/actions/kyverno-wait-ready
-      - name: Test with kuttl
-        shell: bash
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: |
-          set -e
-          ./.tools/kubectl-kuttl test ./test/conformance/kuttl/${{ matrix.tests }} --config ./test/conformance/kuttl/_config/common.yaml
-      - name: Debug failure
-        if: failure()
-        uses: ./.github/actions/kyverno-logs
-
-  chainsaw:
-    runs-on: ubuntu-latest
-    permissions:
-      packages: read
-    strategy:
-      fail-fast: false
-      matrix:
-        config:
-          - name: standard
-            values:
-              - standard
-        k8s-version:
-          - name: v1.25
-            version: v1.25.11
-          - name: v1.26
-            version: v1.26.6
-          - name: v1.27
-            version: v1.27.3
-          - name: v1.28
-            version: v1.28.0
-        tests:
-          - autogen
-          - background-only
-          - cleanup
-          - deferred
-          - events
-          - exceptions
-          - filter
-          - generate/clusterpolicy
-          - generate/policy
-          - generate/validation
-          - mutate
-          - policy-validation
-          - rangeoperators
-          - rbac
-          - reports
-          - validate
-          - verify-manifests
-          - verifyImages
-          - webhooks
-    needs: prepare-images
-    name: chainsaw - ${{ matrix.k8s-version.name }} - ${{ matrix.config.name }} - ${{ matrix.tests }}
```

Comment:
@MariamFahmy98 they didn't match the requirements because their names were not the same, now chainsaw jobs have the same names as the old kuttl ones.

Therefore they will participate to the requirements list.

Reply:
I got it. So where can I find the requirements list?